Makefile

```diff
@@ -65,7 +65,7 @@ dist/$(BINARY_NAME)-%:
 	CGO_ENABLED=0 $(GOARGS) go build -v -ldflags '${LDFLAGS}' -o ${DIST_DIR}/$(BINARY_NAME)-$* ./cmd
 
 .PHONY: image
-image: clean dist/$(BINARY_NAME)-linux-amd64
+image: clean dist/$(BINARY_NAME)-linux-arm64
```

Comment:
I don't think we should commit this, it would be nice if this command automatically picked the right one based on your local chip, but I am unsure of how to do that

Reply:
committed this by mistake, fixing it